### PR TITLE
feat(currency): Add new currencies for fiatconnect

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

Add several new currencies to fully support the current fiatconnect spec [listed here](https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#821-fiattypeenum)

Currencies added:
* Chinese Yuan
* Guinean Franc
* Indian Rupee
* Rwandan Franc
* West African CFA Franc
* Central African CFA Franc
* South African Rand

### Other changes

* Better tests for SelectLocalCurrencies. 
* Render all currencies initially on page

### Tested

ran locally and unit tests.

### How others should test

Go to Settings, select currency.
Verify that the following show up: CNY, GNF, INR, RWF, XOF, XAF

### CODE REVIEW

File: src/localCurrency/SelectLocalCurrency.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ function SelectLocalCurrency() {
           {t('selectCurrency')}
         </Text>
         <FlatList
+          initialNumToRender={30}
```
[the remark this answers is not in the document]
Comment (satish-ravi) [reply src/localCurrency/SelectLocalCurrency.tsx:55]: is this something we have to adjust if we add more currencies and the number goes above 30?

**Review by satish-ravi — APPROVED** [review]

**Review by lisaqyli — APPROVED** [review]

File: src/localCurrency/SelectLocalCurrency.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jh2oman) [reply src/localCurrency/SelectLocalCurrency.tsx:55]: Yes, although a unit test will fail so it will be obvious when we need to update it

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above issue on Latest iOS Test Flight build V:.1.33.0 and updated the status in test rails

**Observation :** Currency symbol does not change on recent transaction activity after selecting different currency and we have raised bug for it

**Bug link:** [Link](https://github.com/valora-inc/wallet/issues/2507)

**Test Rail links:**

**iOS 14:** https://valoraapp.testrail.io/index.php?/tests/view/37171

Devices : iPhone 12(14.7.1)

CC: @jh2oman 

Thanks.!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Test Flight Release build V1.33.0 and updated status in test rails.

**Observations:**

- Currency did not change until user kills the app and reopen the app to view changes in recent transaction activity on homepage.
**Bug:**  [link](https://github.com/valora-inc/wallet/issues/2507)

**Test rail links:**
**iOS 15:** https://valoraapp.testrail.io/index.php?/tests/view/37202

**Devices:** iPhone 13 mini(15.1.1)

**CC:** @jh2oman 

Thanks..!!

**Pull request merged**